build.gradle

```diff
@@ -131,7 +131,7 @@ configure(subprojects.findAll {it.name.endsWith('-mx')}) {
 project(':iso20022-core') {
 	dependencies {
 		// included build
-		api 'com.prowidesoftware:pw-swift-core:SRU2022-9.3.17'
+		api 'com.prowidesoftware:pw-swift-core:SRU2022-9.3.18-SNAPSHOT'
```

Comment:
El cambio no va en Core sino aca, en el metadata extractor default. Y el cambio es dinamico, no hay que alterar el payload del entity, solamente, en caso de que falle el parsing default, intentar fixearlo on the fly, pero solo a fines de que pase el metadata

Reply:
Ok